Formula/quazip.rb

```diff
@@ -12,13 +14,15 @@ class Quazip < Formula
     sha256 "632c10f191326e2afc006c9a065f40af0f5ab8d6b562b4013ecdf77e79ed1eaf" => :high_sierra
   end
 
+  depends_on "cmake" => :build
   depends_on xcode: :build
+
   depends_on "qt"
 
   def install
-    system "qmake", "quazip.pro", "-config", "release",
-                    "PREFIX=#{prefix}", "LIBS+=-lz"
-    system "make", "install"
+    system "cmake", "-DQUAZIP_DIR_NAME=", ".", *std_cmake_args
+    system "cmake", "--build", "."
```

Comment:
```suggestion
    system "make", "CPATH="
```
Should fix the cmath header issues.